base_control/Movement.h

```diff
@@ -0,0 +1,62 @@
+#ifndef MovementClass
+
+#define MovementClass
+
+//MOVEMENT
+class Movement {
+  public:
+    
+    //BACK 
+    Motor B_right;
+    Motor B_left;
+    //FRONT 
+    Motor F_right;
+    Motor F_left;
+    
+    Movement(Motor A, Motor B, Motor C, Motor D);

```

Comment:
back_right_motor, back_left_motor, ... I'm guessing the order of the parameters matter.